### ISSUE TITLE
Fix docker builds

### ISSUE THREAD

**Issue by dhui** [golang-migrate/migrate#207]
```shell
$ docker build .
...
go: google.golang.org/api@v0.0.0-20180910000450-7ca32eb868bf: git -c protocol.version=0 fetch --unshallow -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /go/pkg/mod/cache/vcs/9e62a95b0409d58bc0130bae299bdffbc7b7e74f3abe1ecf897474cc474b8bc0: exit status 128:
	fatal: missing tree object '96ff09559dbf0f63042662c60140f68286c7905f'
	error: remote did not send all necessary objects
verifying google.golang.org/api@v0.3.1/go.mod: checksum mismatch
	downloaded: h1:4mhQ8q/RsB7i+udVvVy5NUi08OU8ZlA0gRVgrF7VFY0=
	go.sum:     h1:6wY9I6uQWHQ8EM57III9mq/AjF+i8G65rmVagqKMtkk=
The command '/bin/sh -c go build -a -o build/migrate.linux-386 -ldflags="-X main.Version=${VERSION}" -tags "$DATABASES $SOURCES" ./cmd/migrate' returned a non-zero code: 1
$
```

On another run, it complains about a different missing tree object:
```shell
$ docker build .
...
go: google.golang.org/api@v0.0.0-20180910000450-7ca32eb868bf: git -c protocol.version=0 fetch --unshallow -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /go/pkg/mod/cache/vcs/9e62a95b0409d58bc0130bae299bdffbc7b7e74f3abe1ecf897474cc474b8bc0: exit status 128:
	fatal: missing tree object 'dcbf3e3c23816703a8f5ebfe6583cc3fc2793526'
	error: remote did not send all necessary objects
go: error loading module requirements
The command '/bin/sh -c go build -a -o build/migrate.linux-386 -ldflags="-X main.Version=${VERSION}" -tags "$DATABASES $SOURCES" ./cmd/migrate' returned a non-zero code: 1
$
```

Seems like this is the underlying issue: https://github.com/golang/go/issues/31399

**Comment by iamtakingiteasy** [comment]
As per issue mentioned, temporary workaround involving golang-migrate in builds might be

```
RUN ( \
      export GO111MODULE=on && cd / && go get google.golang.org/api && \
      cd "$(grep https://code.googlesource.com/google-api-go-client $GOPATH/pkg/mod/cache/vcs/*.info -l | sed 's/\.info$//')" && \
      git fetch -f https://code.googlesource.com/google-api-go-client refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* \
    )
```

**Comment by dhui** [comment]
Once the underlying issue is fixed, I'll push a new release (v4.3.1).
It seems likely that the underlying issue will be resolved soon, so we'll wait instead of using any temp workarounds.

**Comment by dhui** [comment]
[v4.3.1](https://github.com/golang-migrate/migrate/releases/tag/v4.3.1) was successfully built and pushed to [Docker Hub](https://hub.docker.com/r/migrate/migrate/tags)

It looks like the underlying issue has been resolved, at least for our usage.
